src/chart_types/xy_chart/state/chart_state.accessibility.test.ts

```
@@ -52,7 +52,7 @@ describe('test accessibility prop defaults', () => {
       ariaLabelledBy,
     } = getSettingsSpecSelector(state);
     expect(ariaDescription).toBeUndefined();
-    expect(ariaUseDefaultSummary).toBeTrue();
+    expect(ariaUseDefaultSummary).toBeTruthy();
```

Isn't this a `boolean` why the looser assertion?

---

🤔 I'm facing this error in my IDE `any Property 'toBeTrue' does not exist on type 'JestMatchers<boolean>'.ts(2339)`

---

UGH!! 🙄 

This should fix that https://github.com/elastic/elastic-charts/pull/1136. It's a vscode thing not a type issue. You should be able to revert this and just save the file ignoring the error. Test it by running...

```
yarn typecheck:all
```